jobrunner/job_executor.py

```diff
@@ -33,26 +45,86 @@ class JobDefinition:
     allow_database_access: bool  # whether this job should have access to the database
 
 
+class ExecutorState(Enum):
+    PREPARING = "preparing"
+    PREPARED = "prepared"
+    EXECUTING = "executing"
+    EXECUTED = "executed"
+    FINALIZING = "finalizing"
+    FINALIZED = "finalized"
+    GONE = "gone"
+    ERROR = "error"
```

Comment:
The text below refers to this as `ERRORED`. I don't feel strongly which we use as long as it's consistent.

Reply:
This still seems inconsistent to me. Several instances of "ERRORED" below.

Reply:
no ERRORED for me in the branch?